content/en/docs/chart_template_guide/helm_ignore_file.md

```diff
@@ -51,13 +51,15 @@ a[b-d].txt
 temp?
 ```
 
-Some notable differences from .gitignore:
+#### Some notable differences from .gitignore:
 - The '**' syntax is not supported.
 - The globbing library is Go's 'filepath.Match', not fnmatch(3)
 - Trailing spaces are always ignored (there is no supported escape sequence)
 - There is no support for '\!' as a special leading sequence.
 - It does not exclude itself by default, you have to add an explicit entry for `.helmignore`
 
+#### Pitfalls to avoid
+- Do not add `*.tgz` to your list of ignore rules because `helm package` will ignore dependency archives from being packaged in your chart. Instead, narrow down your glob filter or specify exact file names in your rule that does not conflict with any dependency packages
```

Comment:
Did we consider having the package functionality ignore `.helmignore` rules for dependency archives? Not against documenting this behavior in the interim. But the behavior seems wrong / should be fixed.

Reply:
(put the above comment on the issue)

Reply:
It makes sense implementing this in code.